src/pyenphase/updaters/production.py

```diff
@@ -127,6 +148,30 @@ async def update(self, envoy_data: EnvoyData) -> None:
                 production_data
             )
 
+            # get consumption phase data if more then 1 phase is found
+            phase_consumption: dict[str, EnvoySystemConsumption | None] = {}
+            for phase in range(phase_count if phase_count > 1 else 0):
+                consumption: EnvoySystemConsumption | None = (
+                    EnvoySystemConsumption.from_production_phase(production_data, phase)
+                )
+                # exclude None phases that are expected but not actually in production report
+                if consumption:
+                    phase_consumption[list(PhaseNames)[phase]] = consumption
```

Comment:
This looks like it should be a constant instead of creating a list every time

Reply:
For now I've added PHASENAMES constant to const.py based on the enum. Might eliminate the enum altogether in future if it turns out only the list is used.

```python
class PhaseNames(enum.StrEnum):
    PHASE_1 = "L1"
    PHASE_2 = "L2"
    PHASE_3 = "L3"


PHASENAMES = list(PhaseNames)
```